docker-for-windows/index.md

```diff
@@ -793,17 +655,14 @@ Check out these [Docker Cloud topics](/docker-cloud/index.md) to learn more:
 * [Builds and Images](/docker-cloud/builds/index.md)
 * [Swarms in Docker Cloud](/docker-cloud/cloud-swarm/index.md)
 
-Need a direct link to Cloud? [Take me to Docker
-Cloud](https://cloud.docker.com/){: target="_blank" class="_" }.
+Need a direct link to Cloud? [Take me to Docker Cloud](https://cloud.docker.com/){: target="_blank" class="_" }.
 
 ## Where to go next
 
-* Try out the walkthrough at [Get Started](/get-started/).
+* Try out the walkthrough at [Get Started](/get-started/){: target="_blank" class="_"}.
 
 * Dig in deeper with [Docker Labs](https://github.com/docker/labs/) example walkthroughs and source code.
 
-* For a summary of Docker command line interface (CLI) commands,
-see the [Docker CLI Reference Guide](/engine/reference/commandline/docker.md).
+* For a summary of Docker command line interface (CLI) commands, see [Docker CLI Reference Guide](/engine/api.md){: target="_blank" class="_"}.
 
-* Check out the blog post
-  [Introducing Docker 1.13.0](https://blog.docker.com/2017/01/whats-new-in-docker-1-13/).
+* Check out the blog post, [What’s New in Docker 17.06 Community Edition (CE)](https://blog.docker.com/2017/07/whats-new-docker-17-06-community-edition-ce/){: target="_blank" class="_"}.
```

Comment:
That's the latest?  We should have a constant link to the latest version, it's too much work to update IMHO.

Reply:
Let's remove it.